frontend/src/hooks/users.ts

```
@@ -0,0 +1,19 @@
+        await axios.post('http://localhost/api/signup', user);
+    } catch (error: any) {
+        throw error;
+    }
+};
+
+export const fetchUserLogin = async (user: LoginRequest) => {
+    try {
+        const response = await axios.post('http://localhost/api/login', user);
+        return response.data
+    } catch (error: any) {
+        throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)